GitUtils.ps1

```diff
@@ -146,60 +148,87 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
-            dbg 'Getting status' $sw
-            $status = Invoke-Utf8ConsoleCommand { git -c color.status=false status --short --branch 2>$null }
-            if($settings.EnableStashStatus) {
-                dbg 'Getting stash count' $sw
-                $stashCount = $null | git stash list 2>$null | measure-object | Select-Object -expand Count
+            if ($settings.EnableFileStatusFromCache -eq $null) {
+                $settings.EnableFileStatusFromCache = (Get-Module GitStatusCachePoshClient) -ne $null

```

Comment:
Are we sure we want to do this?  It makes it harder to "turn off" this feature if something is misbehaving (or you want to test with it off).  Right now, you'd have to unload (remove-module GitStatusCachePoshClient) and then be careful not to execute a command that would auto-load it again.